server/fleet/app.go

```diff
@@ -527,7 +527,18 @@ type KafkaRESTConfig struct {
 	ProxyHost   string `json:"proxyhost"`
 }
 
-// DeviceAPIFeatures specifies a list of features supported
-// by the current API version. Each field in the struct is
-// meant to be a boolean value.
-type DeviceAPIFeatures struct{}
+// Capability represents a concrete feature of Fleet.
+type Capability string
+
+// The following are the capabilities that Fleet supports. These can be used by
+// the Fleet server, Orbit or Fleet Desktop to communicate that a given feature
+// is supported.
+const (
+	// CapabilityTokenRotation is the ability to rotate and expire host device
+	// tokens over a given period of time.
+	CapabilityTokenRotation Capability = "token_rotation"
```

Comment:
token_rotation is more than anything an example right now so other people can understand the idea behind the PR.

This is a chicken and egg problem, but it will be used in https://github.com/fleetdm/fleet/pull/7779